docs/continuous_replication.md

```diff
@@ -71,3 +71,9 @@ xe vm-param-set blocked-operations:start uuid=<DestinationVM_UUID>
 ### Enable
 
 Manually run the job the first time to check if everything is OK. Then, enable the job. **Now, only the deltas are sent, your initial seed saved you a LOT of time if you have a slow network.**
+
+## HA disabled on replicated VMs
+
+Replicated VMs HA is taken into account by XS/XCP-ng. To avoid the resultant troubles, the property will be removed from the replicated VMs and a tag indicating this change will be added.
```

Comment:
*are taken*

~~the property will be removed~~ -> *HA will be disabled*

Reply:
Maybe add a note explaining that this tag will not be removed automatically (unless we find a way to do it).